src/client/js/components/Admin/SlackIntegration/AccessTokenSettings.jsx

```diff
@@ -1,39 +1,63 @@
-/* eslint-disable no-console */
 import React from 'react';
+import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { toastSuccess } from '../../../util/apiNotification';
 
-function AccessTokenSettings() {
-
+const AccessTokenSettings = (props) => {
   const { t } = useTranslation('admin');
-  function discardHandler() {
-    console.log('Discard button pressed');
-  }
 
-  function generateHandler() {
-    console.log('Generate button pressed');
-  }
+  const onClickDiscardButton = () => {
+    if (props.onClickDiscardButton) {
+      props.onClickDiscardButton();
+    }
+  };
+
+  const onClickGenerateToken = () => {
+    if (props.onClickGenerateToken) {
+      props.onClickGenerateToken();
+    }
+  };
+
+  const showCopiedToaster = () => {
+    toastSuccess(t('slack_integration.copied_to_clipboard'));
+  };
 
   return (
-    <>
-      <div className="form-group row my-5">
-        <label className="text-left text-md-right col-md-3 col-form-label">Access Token</label>
-        <div className="col-md-6">
-          <input className="form-control" type="text" placeholder="access-token" />
+    <div className="row">
+      <div className="col-lg-12">
+
+        <h2 className="admin-setting-header">Access Token</h2>
+
+        <div className="form-group row my-5">
+          <label className="text-left text-md-right col-md-3 col-form-label">Access Token</label>
+          <div className="col-md-6">
+            <CopyToClipboard text={props.accessToken} onCopy={props.accessToken ? showCopiedToaster : null}>
+              <input className="form-control" type="text" value={props.accessToken} readOnly />
```

Comment:
{props.accessToken == null ? (
```suggestion
{props.accessToken == null ? (
  <input className="form-control" type="text" value={props.accessToken} readOnly />
): (
<CopyToClipboard text={props.accessToken} onCopy={showCopiedToaster}>
<input className="form-control" type="text" value={props.accessToken} readOnly />
</CopyToClipboard>
)}
```